src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -415,6 +422,29 @@ def get_az_version_json():
     return versions
 
 
+def get_dependency_versions():
+    versions = {}
+    # Add msal version
+    try:
+        from msal import __version__ as msal_version
```

Comment:
It would work!

By the way, do you consider also printing MSAL EX? :-)

I'm still providing an approval for this part. Other Azure CLI teammates please help review the rest of this PR.

Reply:
> By the way, do you consider also printing MSAL EX? :-)

Let's only add `msal` for now, since the list can go on and on.